Frontend/ui-driver/src/Screens/HomeScreen/Controller.purs

```diff
@@ -128,6 +128,7 @@ instance loggableAction :: Loggable Action where
       -- SelectListModal.ClearOptions -> trackAppActionClick appId (getScreen HOME_SCREEN) "cancel_ride" "clear_options_onclick"
       -- SelectListModal.NoAction -> trackAppActionClick appId (getScreen HOME_SCREEN) "cancel_ride" "no_action"
     GenderBannerModal act -> pure unit
+    AppUpdateBanner act -> pure unit
```

Comment:
@prakharritik can you add proper logs here.

Reply:
Done